lib/snowplow_tracker/emitters/cache.ex

```diff
@@ -0,0 +1,73 @@
+defmodule SnowplowTracker.Emitters.Cache do
+  @moduledoc """
+  Module which implements functions to interact with ETS
+  """
+
+  require Logger
+
+  alias :ets, as: Ets
+
+  @table Application.get_env(:snowplow_tracker, :table)
+  @lock "lock"
+
+  def init(table \\ @table) do
+    PersistentEts.new(
```

Comment:
Specify this option too - https://github.com/michalmuskala/persistent_ets/blob/master/lib/persistent_ets.ex#L63